tests/integ/scala/test_function_suite.py

```diff
@@ -2733,6 +2735,28 @@ def test_lead(session, col_z):
     )
 
 
+@pytest.mark.parametrize("col_z", ["Z", col("Z")])
```

Comment:
Could you also add a test to set ignore_nulls=True? Similar to this one https://github.com/snowflakedb/snowpark-python/blob/2fb6672b9c5851f6098a01e55e64b1d7ffc2fa84/tests/integ/scala/test_window_frame_suite.py#L92 (You can also put that test there)

Reply:
Absolutely! I figured there should be a test for that but couldn't find any in [tests/integ/scala/test_function_suite.py](https://github.com/snowflakedb/snowpark-python/pull/541/files#diff-07654753c6abc298d5834db47798fb90dc448afc67469eaacad27fb1bc6104a0) -- added to the original commit!